aemiers/rancid-tomatillos
# Feature/filter

## Pull Request

### Description

* Is this a feature or a fix?
Feature!

* Where should the reviewer start?
Please start in the dropdown.js document

* What functionalities do these changes affect? Why were these changes made? What was the motivation behind these changes?
Adds a dropdown filter feature with the ability to filter and display movies based on their rating. Cypress tests were added for these additions.

* What outside features or research did you use to implement this fix?
Stack overflow!

* How should this be tested?
Please pull down the branch and see the magic work! An empty state still needs to be added for when filter turns up no search results. I tried to view the empty state with the search bar, but I'm still getting all of the movies showing up when there are no results.

closes #62 
closes #55
